conf/log.conf.d/httpd.portal.conf.example

```diff
@@ -1,56 +1,31 @@
 ### httpd.portal logger ###
-log4perl.rootLogger = INFO, HTTPD_PORTAL
+#log4perl.rootLogger = INFO, PFSYSLOG
```

Comment:
I thought PFSYSLOG made it clearer that this is sent to syslog and not appended to a file.